onnxruntime/test/onnx/main.cc

```diff
@@ -308,6 +308,7 @@ int real_main(int argc, char* argv[], OrtEnv** p_env) {
       {"tf_pnasnet_large", "disable temporarily"},
       {"shrink", "test case is wrong"},
       {"maxpool_2d_precomputed_strides", "ShapeInferenceError"},
+      {"averagepool_2d_precomputed_same_upper", "test case is wrong"},
```

Comment:
Disabling this test because of reason raised here - https://github.com/onnx/onnx/issues/1921. It was passing till now because the pads that got computed by the faulty code somehow happened to match the pad expected by the faulty test case.